hashes/Fedora-Cloud-Base-37-1.7.aarch64-20230330181518.qcow2.sha512

```diff
@@ -1,2 +1,2 @@
 # Fedora-Cloud-Base-37-1.7.aarch64-20230330181518.qcow2
-eedc51c096063333f171c8523a1b70737baa114b9206d87f2b319534bf1d0e3e31a4385169f5c00eb83719291d4b0716251b039bd01d2fa92b70f27f23fb4473 *Fedora-Cloud-Base-37-1.7.aarch64-20230330181518.qcow2
+6cefb0f9fc2c54b2569a07b100d3dee38b254e81dc8c52fdbff71c5f9b8b83c685bfc0978afeb1a2c63358330727bc8ffc651b7d8af3089a266bf496f1e5e523 *Fedora-Cloud-Base-37-1.7.aarch64-20230330181518.qcow2
```

Comment:
The `.sha512` file names should match the OS file name (e.g. `Fedora-Cloud-Base-37-1.7.aarch64-20230417202010.qcow2.sha512`). Same for the file name inside the `.sha512` file itself:

```suggestion
6cefb0f9fc2c54b2569a07b100d3dee38b254e81dc8c52fdbff71c5f9b8b83c685bfc0978afeb1a2c63358330727bc8ffc651b7d8af3089a266bf496f1e5e523 *Fedora-Cloud-Base-37-1.7.aarch64-20230417202010.qcow2
```